rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -95,9 +98,11 @@ impl FmcAliasLayer {
     fn derive_cdi(env: &mut RomEnv, measurements: Array4x12, cdi: KeyId) -> CaliptraResult<()> {
```

Comment:
Should measurements be a reference and zeroized by the caller? (measurements cleared below is a separate shadowed variable)